css-positioning/relative.css

```diff
@@ -0,0 +1,40 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
```

Comment:
better to put position relative here so you don't need it on every pokemon below

Reply:
Got it! Thank you